app/serializers/api/internal/goal_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class Api::Internal::GoalSerializer < ActiveModel::Serializer
+  type :goal
+
+  attributes(
+    :id,
+    :title,
+    :order,
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.